src/components/motion/MotionController.h

```diff
@@ -18,7 +19,17 @@ namespace Pinetime {
         BMA425,
       };
 
-      void Update(int16_t x, int16_t y, int16_t z, uint32_t nbSteps);
+      enum Days {
+        today = 0,
+        yesterday = 1,
+      };
+
+      using step_t = uint32_t;
```

Comment:
I'm really split on this one. On one hand it's nice to have this as a type. On the other, I can't see this type ever being changed (nobody is ever gonna exceed 4 billion steps in a day :P) and it is slower to read and more verbose than `uint32_t`. Also IIRC C++ still will allow using `step_t` and `uint32_t` interchangeably right? (and perhaps any larger/smaller integer type depending on the context). If that's true then we don't get any safety benefits either

Reply:
I was torn when I wrote this too :D

Yeah, it is unlikely to change, since the hardware returns a `uint32_t` through `bma423_step_counter_output()` and since it's an alias, `step_t` is completely interchangeable with `uint32_t`, so having this does not make the code any safer or faster.

But on the other hand, it is more verbose, potentially making the code easier to read (at least for me) that this function returns or expects a type in which steps are stored (even though the function/parameter name should be enough), much like how `size_t` is used for most if not all functions where a container size is used.

At least that was my rationale when I wrote this, but I can change it back to `uint32_t` if needed (or move it out of the `MotionController` class).

Reply:
I think when it come verboseness the advantage I see is instead of having to reference `Controllers::MotionController::step_t` instead of `uint32_t`. To me that's easier to read, with the operations available on the type being clearer rather than having to look it up and go "oh, it's just an unsigned int, right". But your point about the semantics of the variable being clearer is fair.

I don't know, maybe someone else will have an opinion? I don't strongly oppose it either way

Reply:
It's just a count, right? Maybe keep it simple. I don't see, what problem would be solved, or is there any critical code, where confusion must not happen (e.g. temperature, where we handle values with different units)?

Reply:
Yeah, it's just a counter, I just didn't feel important to know the real underlying type everywhere in the code.

Even **if** `step_t` makes sense, it definitely shouldn't be defined within `MotionController`, so I'm removing it